samples/highcharts/chart/renderto-jquery/demo.js

```diff
@@ -1,15 +0,0 @@
-var $container = $('<div>').appendTo(document.body);
```

Comment:
This sample is referred to from the API docs. If we remove the sample, we must remove the reference too: https://github.com/highcharts/highcharts/blob/samples/delete-jquery/ts/Core/Options.ts/#L1754-L1755